File: java/gpt/P25.java
import java.util.Random;

public class P25 {
    public static int getMonthlySales(int month) {
        Random r = new Random();
        return r.nextInt(100001); // 0..100000
    }

    public static void main(String[] args) {
        int[] months = new int[12];
        for (int i = 0; i < 12; i++) months[i] = getMonthlySales(i);
        for (int i = 0; i < 12; i++) System.out.println("Month " + (i+1) + ": " + months[i]);
        for (int q = 0; q < 4; q++) {
            int sum = 0;
            for (int j = q*3; j < q*3+3; j++) sum += months[j];
            System.out.println("Quarter " + (q+1) + " sum: " + sum);
        }
    }
}

File: javascript/gpt/p25.js
// p25: call getMonthlySales 12 times, sum quarterly and print
function getMonthlySales(i) {
  return Math.floor(Math.random() * 100001); // 0..100000
}
const months = [];
for (let i = 0; i < 12; i++) months.push(getMonthlySales(i));
months.forEach((m, idx) => console.log(`Month ${idx + 1}: ${m}`));
const qSums = [];
for (let q = 0; q < 4; q++) {
  const s = months.slice(q * 3, q * 3 + 3).reduce((a, b) => a + b, 0);
  qSums.push(s);
}
qSums.forEach((s, i) => console.log(`Quarter ${i + 1} sum: ${s}`));
